Maths/test/WhileLoopFactorial.test.js

```diff
@@ -0,0 +1,21 @@
+import { factorialize } from '../WhileLoopFactorial'
+
+test('Testing on 3!', () => {
+  const three_factorial = factorialize(3)
```

Comment:
Your identifiers should be in camel case. You also don't need the variables (you should just do `expect(factorialize(3)).toBe(6)`).